tigerpath/forms.py

```diff
@@ -16,8 +16,7 @@ def create_major_choices():
     majors = Major.objects.order_by('name').values_list('id', 'name')
     supported_majors = tuple(majors.filter(supported=True))
     unsupported_majors = tuple(majors.filter(supported=False))
-    return (('Fully supported', supported_majors),
-            ('Not fully supported yet - only AB/BSE requirements supported', unsupported_majors))
+    return (('', supported_majors), ('', unsupported_majors))
```

Comment:
Since all the majors will be supported, I don't think we need the supported/unsupported filtering anymore. Would be better if we just returned the majors directly

Reply:
Yeah, you're right!

Reply:
Fixed.